requirements.txt

```diff
@@ -0,0 +1,5 @@
+faiss-cpu==1.6.3
```

Comment:
@ManuelAlvarezC @antonpolishko 
I am using a conda virtual environment and pip install did not work for me.  I had to use `conda install faiss-cpu -c pytorch`.  Is this something for which we should include a note in the ReadMe (e.g. if using conda, comment out this line and install with...) or will we address in the `requirements.txt` / future packaging configuration ?